app/src/main/java/co/nimblehq/compose/crypto/ui/theme/Dimension.kt

```diff
@@ -20,4 +20,9 @@ object Dimension {
     val Dp45 = 45.dp
     val Dp52 = 52.dp
     val Dp60 = 60.dp
+
+    // Card shadow
+    val shadowBorderRadius = 12.dp
+    val shadowBlurRadius = 24.dp
```

Comment:
Should we put it in `Shape` instead 🤔

Reply:
@Wadeewee I think no need. These fields are supposed for dimensions only.